packages/sdk-components-react-radix/src/checkbox.ws.ts

```diff
@@ -146,7 +146,7 @@ export const metaCheckboxIndicator: WsComponentMeta = {
 
 export const propsMetaCheckbox: WsComponentPropsMeta = {
   props: propsCheckbox,
-  initialProps: ["id", "className", "checked", "name", "required"],
+  initialProps: ["id", "className", "name", "value", "required", "checked"],
```

Comment:
What does "value" mean in checkbox?

Reply:
same as in native checkbox https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value